codacy/values.yaml

```diff
@@ -273,6 +273,10 @@ worker-manager:
       tolerations: []
       nodeSelector: {}
 
+    codacy:
+      worker:
+        image: 5.56.0
```

Comment:
this needs to be updated automatically in the pipeline when there is a release

Reply:
do you mean that this would be another helm --set ?
does circleci support explicitly asking for parameter values? so that when you cut the release, you can input the value of the worker image that you want to use? I can not find that anywhere :/

Reply:
no.. with the ytool. This needs to have the same value as the engine.

Reply:
got it.